docs/vsphere-golden-image.md

```diff
@@ -158,13 +165,21 @@ where `<path/to/unattend.xml>` is the path to the customized answer file.
 Note: There is [a limit](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep--generalize--a-windows-installation#limits-on-how-many-times-you-can-run-sysprep)
 on how many times you can run the `sysprep` command.
 
-## 7. Set up the virtual machine golden image
+## 7. Set up the virtual machine template
```

Comment:
Has your PR merged that uses the template?

Reply:
No, just missing a LGTM

See https://github.com/openshift/windows-machine-config-operator/pull/544

Reply:
#544 merged